tox.ini

```diff
@@ -25,9 +25,10 @@ passenv =
   POSTGRES_TEST_*
   PYTEST_ADDOPTS
 commands =
-  {envpython} -m pytest --cov=core -m profile_postgres {posargs} test/integration
   {envpython} -m pytest --cov=core {posargs} tests/functional
   {envpython} -m pytest --cov=core {posargs} tests/adapter
+  {envpython} -m pytest --cov=core -m profile_postgres {posargs} test/integration
```

Comment:
Nothing important, but I'm curious why the test run-order changed?

Reply:
Given the test/integration tests will continue to fail on this branch, I moved them to run last so that errors from `tests/functional` would be visible in CI logs